demo/demo-tekton.sh

```diff
@@ -307,7 +343,7 @@ if [ "${NO_HUMAN}" == false ]; then
           * ) echo "I'm not sure what $a means, please give 1 or 2" >&2 ;;
        esac
     done
-elif [ "${NO_HUMAN}" == true ]; then
+else
```

Comment:
I would stay explicit here. else means we would need to add comparison to check if the NO_HUMAN var exists and so on...

Reply:
But its defined in the defaults (so it is always available), and the user can not change its value (only the script changes it to true). Do you still prefer to be explicit?